dsa-23au/java-dsa/arrays-links/src/main/java/dev/codewithfriends/LinkedListWrapper.java

```diff
@@ -166,11 +166,4 @@ public boolean containsAll(Collection c) {
     public Object[] toArray(Object[] a) {
         return new Object[0];
     }
-
```

Comment:
[Assignment and Co-Creation Week 04](https://github.com/TheEvergreenStateCollege/upper-division-cs/blob/main/dsa-23au/notes/docs/week-04/Assignment-04.md#assignment-and-co-creation-week-04)
Perfect work